docs/en/multi_modal/deepseek_vl2.md

```diff
@@ -0,0 +1,43 @@
+# DeepSeek-VL2
+
+## Introduction
+
+DeepSeek-VL2, an advanced series of large Mixture-of-Experts (MoE) Vision-Language Models that significantly improves upon its predecessor, DeepSeek-VL.
+DeepSeek-VL2 demonstrates superior capabilities across various tasks, including but not limited to visual question answering, optical character recognition, document/table/chart understanding, and visual grounding.
+
+LMDeploy supports [deepseek-vl2-tiny](https://huggingface.co/deepseek-ai/deepseek-vl2-tiny), [deepseek-vl2-small](https://huggingface.co/deepseek-ai/deepseek-vl2-small) and [deepseek-vl2](https://huggingface.co/deepseek-ai/deepseek-vl2) in PyTorch engine.
+
+## Quick Start
+
+Install LMDeploy by following the [installation guide](../get_started/installation.md).
+
+### Prepare
+
+When deploying the **DeepSeek-VL2** model using LMDeploy, you must install the official GitHub repository and related 3-rd party libs. This is because LMDeploy reuses the image processing functions provided in the official repository.
+
+```
+pip install git+https://github.com/deepseek-ai/DeepSeek-VL2.git --no-deps
+pip install attrdict
```

Comment:
better to keep all dependencies in one line `pip install attrdict timm 'transformers<=4.48.0'`